sklearn/utils/_show_versions.py

```diff
@@ -71,15 +73,20 @@ def get_version(module):
 
 
 def show_versions():
-    "Print useful debugging information"
+    """Print useful debugging information"""
 
     sys_info = _get_sys_info()
     deps_info = _get_deps_info()
 
-    print('\nSystem:')
+    print('\nSystem')
+    print('------')
```

Comment:
I wanted to do that too long ago, but we decided against it (I think it was @rth?)

Users are asked to copy/paste this in GitHub and that will mess with the markdown :(

Reply:
oh :(